### ISSUE TITLE
TOS "click here" link needs to be renamed

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#3978]
## Area or Component
Patient Experience - Terms of Service

## Quick Summary
One of the links in the terms of service is labelled simply as "click here", but this doesn't give enough information to users about what the link is.

## Accessibility Violation
The element's accessible name is missing or is not appropriate for the element's purpose. https://axe.deque.com/rule-help/en/aria-name-missing-incorrect 

See also the [links section of the Deque course.](https://dequeuniversity.com/class/semantic-structure/links/link-text#links-meaningful-text-alone:~:text=End%20code.-,The%20purpose%20of%20each%20link%20SHOULD%20be%20able%20to%20be%20determined%20from%20the%20link%20text%20alone.,-Screen%20reader%20users)

## Action Requested
Change the link text to:

> If you would like more information about the application of the Privacy Act at CDC, visit the [Health and Human Services website](https://www.hhs.gov/foia/privacy/index.html).

Spanish translation:
> Si desea obtener más información sobre la aplicación de la Ley de Privacidad en los CDC, visite el [sitio web del Departamento de Salud y Servicios Humanos](https://www.hhs.gov/foia/privacy/index.html).

Note: This change was approved by the CDC’s Office of General Counsel through Mike Judd. 

## Source Code (optional)
https://github.com/CDCgov/prime-simplereport/blob/c3e6b2dc330021e425e01657eaee613c51f9e11c/frontend/src/patientApp/timeOfTest/TermsOfService.tsx#L58-L58

## Original Deque Issue
aria-name-missing-incorrect | The element's accessible name is missing or is not appropriate for the element's purpose. | "click here" is not an accessible enough name; should describe where the link goes | Name: The element's name is missing or incorrect | https://axe.deque.com/rule-help/en/aria-name-missing-incorrect | critical | Yes | No | Interactive Elements | ["body > div#root:nth-of-type(1) > div.bg-base-lightest.minh-viewport:nth-of-type(1) > div.bg-base-lightest:nth-of-type(3) > main.patient-app.padding-bottom-4.bg-base-lightest.padding-top-05:nth-of-type(1) > form.grid-container.maxw-tablet.usa-prose:nth-of-type(1) > button#tos-consent-button.usa-button:nth-of-type(1)"] | The element's accessible name is missing or is not appropriate for the element's purpose. | <button type="button" class="usa-button" id="tos-consent-button" data-axe-pro-tab-stop-index="8">I agree</button> | wcag2a,wcag412b | rnv7@cdc.gov | Patient Self Reg - Terms of Service | https://stg.simplereport.gov/app/register/9C7BB |   | 2022-07-13T21:04:27.207Z
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --

**Comment by lauraykerr** [comment]
Recommended copy: 
`If you would like more information about the application of the Privacy Act at CDC, [visit the Centers for Medicare & Medicaid Services website](https://www.cms.gov/Research-Statistics-Data-and-Systems/Computer-Data-and-Systems/Privacy/PrivacyActof1974.html).`

**Comment by lauraykerr** [comment]
@emmastephenson Have you cleared changes with legal before? Happy to kick that off or defer to you and @jayna-SkylightDigital

**Comment by emmastephenson** [comment]
@lauraykerr I have not, if you know the process it'd be great if you could kick it off!

**Comment by lauraykerr** [comment]
@zdeveloper Here's another for Better Data! Final content, which has legal approval, is in the **Action Requested** section of the description above.

**Comment by lauraykerr** [comment]
Ticket [#353](https://github.com/CDCgov/prime-simplereport-site/pull/353) resolves this issue for static site TOS.

) resolves this issue for static site TOS.

**Comment by mehansen** [comment]
hi @lauraykerr! I updated the code to reflect the new copy and the link label is the native text of the link between the `<a>` and `</a>` elements. can you please confirm that this screenshot matches what we want? the label now says "Health and Human Services website". if not I can give it a different label.
<img width="1552" alt="Screen Shot 2022-09-30 at 3 45 00 PM" src="https://user-images.githubusercontent.com/6401323/193369152-365a24fb-cede-4797-88cd-f5d3dbda9dd4.png">

fwiw, the [static site label for the same link](https://github.com/CDCgov/prime-simplereport-site/blob/0d872bd886b133ad6df52385c13c9c9cd501464a/pages/tos.md?plain=1#L39) is "visit the Health and Human Services website". not sure if we want them to be exactly the same.

additionally, who can I reach out to for copy for the Spanish version of the ToS?

**Comment by lauraykerr** [comment]
@mehansen The link text looks great, thank you! I'll change the static site TOS to match this. 

I can also get the Spanish version. To confirm, we have an existing Spanish version but will just need translations of the updated text?

**Comment by mehansen** [comment]
@lauraykerr thank you!

we do have an existing Spanish version, just need a translation for "If you would like more information about the application of the Privacy Act at CDC, visit the [Health and Human Services website]." so I can swap out that sentence

screenshot of what it currently looks like, inc label of link (in bottom right):
![Screen Shot 2022-10-03 at 12 15 57 PM](https://user-images.githubusercontent.com/6401323/193659955-2b0c3f02-4f60-44b9-8652-cbd866a08a37.png)

**Comment by lauraykerr** [comment]
Sent to translation team through Holly Wilson and Dana Pitts this morning (10/13)

**Comment by lauraykerr** [comment]
@mehansen @zdeveloper Spanish translation added to the description!
